### DISCUSSION TITLE
initialCache: false setting makes two requests in turn

### DISCUSSION THREAD

**Discussion by rudimadima** [nuxt/nuxt#17179] [Questions]
```vue
<template>
  <div> {{ data }} </div>
</template>

<script setup lang="ts">
const { data, error } = await useAsyncData(
  `test`,
  () => {
    console.log('fetching...');
    return Promise.resolve('hi!');
  },
  {
    initialCache: false,
  }
);
</script>
```
When the initialCache setting is set to false, the first time the site is opened, data is received first on the server, and then again in the browser.

I.e., instead of one request, two identical requests are sent in turn: first from the server, then from the browser.

How to avoid this?

I need to disable caching of the request response so that when navigating through the pages of the site, data is not taken from the cache. But I don't need the request to happen twice the first time I open the site.

**Comment by Flanker72** [comment]
@danielroe seems we need you help, because I have the same issue.

**Reply by danielroe** [reply]
Would you provide a reproduction and open an issue? I can't reproduce with the original code: https://stackblitz.com/edit/github-udcw28

**Reply by Flanker72** [reply]
Created an issue https://github.com/nuxt/framework/discussions/6236 with the reproduction link attached.
